Classes/JSONAPIResourceParser.m

```diff
@@ -205,7 +205,7 @@ + (void)set:(NSObject <JSONAPIResource> *)resource withDictionary:dictionary {
             
         } else {
             id value = [attributes objectForKey:[property jsonName]];;
-            if (nil == value) {
+            if ((id)[NSNull null] == value) {
```

Comment:
@christianklotz Should be be keeping the check for `nil` but also check for `NSNull`?